actions/plan/action.yml

```
@@ -67,6 +72,7 @@ runs:
         PATH_TO_ARCHIVE: ${{ steps.setup.outputs.artifacts_dir }}
 
     - working-directory: ${{ fromJson(inputs.config).workspace_path }}
+      if: ${{ inputs.comment_plan == true }}
```

I'm out of ideas atm. I've tried this (doesn't work for `true`) and 

```suggestion
      if: ${{ inputs.comment_plan }}  # doesn't work for `false`
```

```suggestion
      if: inputs.comment_plan  # doesn't work for false
```

Maybe related to https://github.com/actions/runner/issues/1173?

PR used to test, although it's hard to line up the commits here and there: https://github.com/Brightspace/d2l-eventwebhooks/pull/124

---

6aa6137a4ff2f28d05d6c5d4cd30353a3bdc1c88 did the trick. Thanks! (See https://github.com/Brightspace/d2l-eventwebhooks/pull/124#commits-pushed-9217668)